keyboards/converter/att56k_usb/info.json

```
@@ -0,0 +1,136 @@
+        {"label":"U", "x":9.75, "y":4},
+        {"label":"I", "x":10.75, "y":4},
+        {"label":"O", "x":11.75, "y":4},
+        {"label":"P", "x":12.75, "y":4},
+        {"label":"[", "x":13.75, "y":4},
+        {"label":"\\", "x":14.75, "y":4},
+        {"label":"Return", "x":15.75, "y":4, "w":1.5, "ks":[ [0,0], [1.5,0], [1.5,2], [0.25,2], [0.25,1], [0,1], [0,0] ]},
+        {"label":"Back Tab", "x":17.5, "y":4},
+        {"label":"Insert", "x":18.5, "y":4},
+        {"label":"Delete", "x":19.5, "y":4},
+        {"label":"7", "x":20.75, "y":4},
+        {"label":"8", "x":21.75, "y":4},
+        {"label":"9", "x":22.75, "y":4},
+        {"label":"Tab", "x":23.75, "y":4},
+        {"label":"Jump", "x":0, "y":5},
+        {"label":"Erase EOF", "x":1, "y":5},
+        {"label":"Caps Lock", "x":2.25, "y":5, "w":1.75},
+        {"label":"A", "x":4, "y":5},
+        {"label":"S", "x":5, "y":5},
+        {"label":"D", "x":6, "y":5},
+        {"label":"F", "x":7, "y":5},
+        {"label":"G", "x":8, "y":5},
+        {"label":"H", "x":9, "y":5},
+        {"label":"J", "x":10, "y":5},
+        {"label":"K", "x":11, "y":5},
+        {"label":"L", "x":12, "y":5},
+        {"label":";", "x":13, "y":5},
+        {"label":"'", "x":14, "y":5},
+        {"label":"{", "x":15, "y":5},
+        {"label":"Up", "x":18.5, "y":5},
+        {"label":"4", "x":20.75, "y":5},
+        {"label":"5", "x":21.75, "y":5},
+        {"label":"6", "x":22.75, "y":5},
+        {"label":"-", "x":23.75, "y":5},
+        {"label":"Print", "x":0, "y":6},
+        {"label":"Play", "x":1, "y":6},
+        {"label":"Shift", "x":2.25, "y":6, "w":1.25},
+        {"label":"<>", "x":3.5, "y":6},
+        {"label":"Z", "x":4.5, "y":6},
+        {"label":"X", "x":5.5, "y":6},
+        {"label":"C", "x":6.5, "y":6},
+        {"label":"V", "x":7.5, "y":6},
+        {"label":"B", "x":8.5, "y":6},
+        {"label":"N", "x":9.5, "y":6},
+        {"label":"M", "x":10.5, "y":6},
+        {"label":",", "x":11.5, "y":6},
+        {"label":".", "x":12.5, "y":6},
+        {"label":"/", "x":13.5, "y":6},
+        {"label":"Shift", "x":14.5, "y":6, "w":2.75},
+        {"label":"Left", "x":17.5, "y":6},
+        {"label":"No", "x":18.5, "y":6},
+        {"label":"Right", "x":19.5, "y":6},
+        {"label":"1", "x":20.75, "y":6},
+        {"label":"2", "x":21.75, "y":6},
+        {"label":"3", "x":22.75, "y":6},
+        {"label":"Enter", "x":23.75, "y":6, "h":2},
+        {"label":"Min", "x":0, "y":7},
+        {"label":"WS Ctrol", "x":1, "y":7},
+        {"label":"Rese", "x":2.25, "y":7, "w":1.5},
+        {"label":"Alt/Ctrk", "x":4.75, "y":7, "w":1.5},
+        {"label":"Space", "x":6.25, "y":7, "w":7},
+        {"label":"Alt/Ctrl", "x":13.25, "y":7, "w":1.5},
+        {"label":"Enter", "x":15.75, "y":7, "w":1.5},
+        {"label":"Down", "x":18.5, "y":7},
+        {"label":"No", "x":20.75, "y":7},
+        {"label":"0", "x":21.75, "y":7, "w":2},
+        {"label":".", "x":23.75, "y":7}
```

You have an extra key here.

```suggestion
        {"label":"0", "x":20.75, "y":7, "w":2},
        {"label":".", "x":22.75, "y":7}
```